code401-challenges/src/main/java/code401/challenges/linkedlist/LinkedList.java

```diff
@@ -172,14 +169,30 @@ public int knthFromEnd(int value){
         return -1;
     }
 
-
-    public static void main (String[] args){
-        LinkedList test = new LinkedList();
-        test.append(1);
-        test.append(3);
-        test.append(8);
-        test.append(2);
-
-        System.out.println(test.knthFromEnd(1));
+    public static Node mergeLists(LinkedList one, LinkedList two){
+        if(one.head == null){
+            return two.head;
+        }else if(two.head == null){
+            return one.head;
+        }else{
+            Node oneCurrent = one.head;
+            Node twoCurrent = two.head;
+            LinkedList mergedList = new LinkedList();
+            while(oneCurrent != null || twoCurrent != null){
+                if(oneCurrent == null){
```

Comment:
I don't think this conditional will ever happen. If `oneCurrent == null` then the `while` loop won't run again